README.md

```diff
@@ -9,7 +9,7 @@ There are currently 1 HPE Nimble Storage Collections
 
 Install the HPE Nimble Storage array collection on your Ansible management host.
 
-- Download the collection from here [\[ansible-collection-21-06-2020\]](https://confluence.eng.nimblestorage.com/download/attachments/102956388/hpe-nimble-1.0.0.tar.gz?version=2&modificationDate=1592618015619&api=v2)
+- Download the collection from here [Nimble-Ansible-collection-builds](https://confluence.eng.nimblestorage.com/display/IFP/Ansible+Storage+Modules#AnsibleStorageModules-AnsibleCollectionBuilds:)
```

Comment:
HPE Cyber will not tolerate internal links being posted on GitHub. They will request the repository being deleted and recreated if they see this.

Reply:
Thanks Michael,
i was not aware.. will remove the link and just say. Download from  confluence

Reply:
do not say "download from confluence".  This is for external consumption.

Reply:
will remove